build.gradle.kts

```diff
@@ -9,6 +11,8 @@ plugins {
     id("org.gradle.wrapper-upgrade") version "0.11.1"
 }
 
+group = "org.gradle"
```

Comment:
Please use `com.gradle`, like we do for CCUD, etc. (given this is GE related).

Reply:
Done in 20e1a85d9309aa7e2a9028d1dd53c7246611bb90